src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3310,12 +3315,31 @@ static bool isConditionalAccessInstancePresentInChildren(IOperation operation)
                     {
                         return true;
                     }
+                    else if (operation is InvalidOperation invalidChild)
```

Comment:
>else if (operation is InvalidOperation invalidChild) [](start = 20, length = 52)

I am not sure why do we need to do this this check inside the loop. Are we checking this at the beginning of the function?

Reply:
Typo, that should have been `enumerator.Current`. Adding a test to demonstrate.

---
In reply to: [577691937](https://github.com/dotnet/roslyn/pull/51206#discussion_r577691937) [](ancestors = 577691937)